### DISCUSSION TITLE
Touring agent #1 - Gather and publish information about the productions I represent on the touring market

### DISCUSSION THREAD

**Discussion by fjjulien** [pair-cg/general-contributions#63] [User Stories]
# Motivation
As a touring agent, I represent different dance and theatre companies who wish to sell their productions on the touring market. But before I even make a first sales pitch, I need to gather all possible information that I might want to use in my sales pitch. Then, I must also ensure that information is readily available in online directories that potential buyers might consult.

# 1. Need:
Part 1: Information about the dance and theatre companies I represent relevant for sales pitches. 
Part 2: Raise awareness about the production to potential buyers.

# 2. Action:

Part 1: Collect information about the dance and theatre companies I represent relevant for sales pitches.

**Company Information:** 
1. When was the company founded? 
(How long has the company been operating can be inferred from when it was founded)

2. What is the artistic vision of the company? (This information is usually taking the form of a main textual declarative statement along, sometimes with additional textual narratives to highlight particular aspects or features of the artistic vision or aspects of the artistic practice of the person holding the artistic vision)

3. Who are the founder(s) of the company? 

4. Have the founder(s) of the company won any awards? If so, which ones?  

5. Who is the current artistic director(s)? 

6. What are some notable productions directed / choreographed by the current artistic director(s) ? 
   (notability indicators can include awards/distinctions received, the total number of individual performances, the number of times it was remounted, the number of cities it has toured in, how long it has been on the touring market, or any quantitative indicator of artistic significance vis-a-vis the company's artistic diretion)?

**Production information:**

**_If it's a production that hasn't been premiered yet:_**

**Content:**
1. What is the provisional title?
2. What is the provisional synopsis.
3. What is the development history of the work? (Has it had a public reading at a festival or industry event?)
4. What's the artistic impetus? 
5. What's driving the creative process for the work? 
6. Who are the confirmed contributors? 
7. Who are the confirmed performers?
8. How does the work resemble previous productions of the company?
9. How does the work differ from previous productions of the company?
10. What is the ideal start date and end date for the tour? (day/month/year)
11. What geographical area or location does the artistic company want to tour the production? 

**Artifacts**
1. What promotional material is available? (hard / digital copies or links to digital copies of photos, video trailer, a full-length videoe, etc.)

**_If it's a production that has been premiered_**

**Content:**
1. What is the title?
2. What is the synopsis.
3. What is the development history of the work? (Has it had a public reading at a festival or industry event?)
4. What's the artistic impetus? 
5. What's driving the creative process for the work? 
6. Who are the confirmed contributors? 
7. Who are the confirmed performers?
8. How does the work resemble previous productions of the company?
9. How does the work differ from previous productions of the company?
10. What is the ideal start date and end date for the tour? (day/month/year)
11. What geographical area or location does the artistic company want to tour the production? 
12. Where did it premiere?
13. What (day/month/year) did it premiere?

**Artifacts**
1. What promotional material is available? (hard / digital copies or links to digital copies of photos, video trailer, a full-length videoe, etc.)

**Feedback**
1. How was the production received by the public, by peers, by the media? (need some measurable indicators)
- Audience feedback (provided as emails, social media responses, blog post comments, etc.)
- Critic feedback (reviews, recommendations, previews, critic's picks)

Part 2: Publish the information online in directories (maintained by presenters and field associations) so that potential buyers can find and buy the production.

# 3. Desired Outcome:

To get all the information that I need and that the relevant people become aware of the productions I am representing. (spend less time gathering, populating, and updating information. And more time prospecting potential buyers. Please.)

# 4. Perspective:

Touring Agent. 

# 5. Stakeholder Group:

Touring Agent
Potential Buyers / Presenters
Arts Companies / Artists
Producing company 
Buyers on the performing arts  touring market

# 6. Additional info:

Stakeholders:
Potential buyers on the touring market include specialized presenters (other producing companies that occasionally present works other than theirs) and multidisciplinary presenters (organizations that manage a performing arts centre and that program series of performances by touring companies). The two buyers groups have different levels knowledge of the performing arts genres I represent as an agent. Therefore, they need different kinds of information. The specialized presenters in the domestic market are usually well aware of the companies I represent. If the show has already been premiered, they may have heard of it. In this particular case, I need qualitative information that will enable me to describe the artistic process and the aesthetics of the production. If the show hasn't been premiered and is still in development, then I need to have access to the latest information about all confirmed contributors and cast members. When selling the show to multi-disciplinary presenters, I can't presume a pre-existing knowledge of the production or of the company. I need to be able to quickly describe (and differentiate) the artistic vision of the company, highlight any past awards received, and then I can share information about the production that I wish to sell. I will likely begin by sharing a short synopsis and then I will describe particular features of this production that might set it appart from others.

Informational needs (Internal dialogue): 
1. Really, you don't have this information on your own website? Let's get this fixed.
2. I will populate the information in online directories maintained by presenters and disciplinary associations. But each one asks for slightly different information. And why do they have different maximum word counts? So I have to craft three descriptions of the production: a short, a medium and a longer one. 
3. Frédéric has told me I should put this on Wikidata, but I don't know how it works and it really doesn't look appealling, so I'll skip this one.
4. "Shit! You've lost one of your main cast members, because he accepted a film gig!?! The brat!" This means I have to back in each directory and update the information about the cast. What a waste of time.
5. "Wonderful, you won a Dora award!" This means I once again have to go back in each directory to add this information. This feels slightly less like a waste of time, but it still is.

**Comment by mariel-marshall** [comment]
@fjjulien this is really great contribution. I made a few proposed edits above. Feel free to look over and see if you agree!

**Reply by fjjulien** [reply]
Thanks for your edits @mariel-marshall. I conflated all productions notability indicators under the same question. It seemed more logical to group them in this way.

**Reply by fjjulien** [reply]
"How long has the company been operating can be inferred from when it was founded"

Founding date is an important information about a producing company. At the same time, it is a fuzzy concept.  Based on my experience collecting and populating organization information over Wikidata, there can be more than one value for an organization's "start date". 

Most often, the organization has a fuzzy beginning when a group of people makes a decision to create an organization or else hold a first event, which is deemed foundational. Quite often, the exact date of this foundational moment is very hard to find. It may have never been documented in writing and organizations themselves may not even be able to give you a specific date.

Then an organization may incorporate or register as a juridical person in a given jurisdiction (i.e. incorporating as a not-for-profit organization according to the laws of Ontario). This other foundational moment is more accurately documented and a specific date can easily be found.

Many times, these two foundational moments tend to be the same: the memory of the first foundational moment fades and the corporation foundation takes over as the sole foundational moment. Other times both information co-exist in the history of the organization.

**Reply by drosu** [reply]
This is an excellent point ! The framework should for flexibility around specifying/communicating this information. There is uncertainty related to the other types of information as well, i.e., year a creative work was first published and/or performed, as historical documents can contain multiple conflicting accounts, or no accounts at all (and the year of first publication/performance is a guess based on historical / literary research).
